src/core/namespace.c

```diff
@@ -204,6 +205,16 @@ static const MountEntry protect_control_groups_yes_table[] = {
         { "/sys/fs/cgroup",      MOUNT_READ_ONLY,         false  },
 };
 
+/* ProtectControlGroups=private table. */
+static const MountEntry protect_control_groups_private_table[] = {
+        { "/sys/fs/cgroup",      MOUNT_PRIVATE_CGROUP2FS, false, .read_only = false, .nosuid = true, .noexec = true, .options_const = "nsdelegate", .flags = MS_NOSUID|MS_NOEXEC|MS_NODEV  },
```

Comment:
Not a matter, but .flags is unused by mount_private_apivfs().

Reply:
Yea I can remove flags here and add a comment